raft/raft.cpp

```diff
@@ -867,27 +867,44 @@ raft::get_leader()
 void
 raft::initialize_storage_from_log(std::shared_ptr<bzn::storage_base> storage)
 {
+    LOG(info) << "Initializng storage from log entries";
     for (const auto& log_entry : this->raft_log->get_log_entries())
     {
         if(log_entry.entry_type == bzn::log_entry_type::database)
         {
-            const auto command = log_entry.msg["cmd"].asString();
-            const auto db_uuid = log_entry.msg["db-uuid"].asString();
-            const auto key = log_entry.msg["data"]["key"].asString();
-            if (command == "create")
+            bzn_msg msg;
+
+            if(!msg.ParseFromString(boost::beast::detail::base64_decode(log_entry.msg["msg"].asString())))
```

Comment:
space after if

Reply:
SON OF A... ok ok, I missed this one .. my bad... As punishment for my transgression I made the change to all the other offending if statements.